time-to-live.md

```diff
@@ -0,0 +1,167 @@
+
+You can use TTL together with the [JSON type](/data-type-json.md). For example:
+
+```sql
+CREATE TABLE orders (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    order_info JSON,
+    created_at DATE AS (JSON_EXTRACT(order_info, '$.created_at')) VIRTUAL
+) TTL = `created_at` + INTERVAL 3 month;
+```
+
+## TTL job
+
+For each table with a TTL attribute, TiDB internally schedules a background job to clean up expired data. You can customize the execution period of the job by setting the [`tidb_ttl_job_run_interval`](/system-variables.md#tidb_ttl_job_run_interval-new-in-v650) global variable. The following example sets the background cleanup job to run once every 24 hours:
+
+```sql
+SET @@global.tidb_ttl_job_run_interval = '24h';
+```
+
+To disable the execution of the TTL job, in addition to setting the `TTL_ENABLE='OFF'` table option, you can also disable the execution of the TTL job in the entire cluster by setting the [`tidb_ttl_job_enable`](/system-variables.md#tidb_ttl_job_enable-new-in-v650) global variable:
+
+```sql
+SET @@global.tidb_ttl_job_enable = OFF;
+```
+
+In some scenarios, you might want to allow the TTL job to run only in a certain time window. In this case, you can set the [`tidb_ttl_job_schedule_window_start_time`](/system-variables.md#tidb_ttl_job_schedule_window_start_time-new-in-v650) and [`tidb_ttl_job_schedule_window_end_time`](/system-variables.md#tidb_ttl_job_schedule_window_end_time-new-in-v650) global variables to specify the time window. For example:
+
+```sql
+SET @@global.tidb_ttl_job_schedule_window_start_time = '01:00 +0000';
+SET @@global.tidb_ttl_job_schedule_window_end_time = '05:00 +0000';
+```
+
+The preceding statement allows the TTL job to be scheduled only between 1:00 and 5:00 UTC. By default, the time window is set to `00:00 +0000` to `23:59 +0000`, which allows the job to be scheduled at any time.
+
+## Compatibility with tools
+
+As an experimental feature, the TTL feature is not compatible with data import and export tools, including BR, TiDB Lightning, and TiCDC.
+
+## Limitations
+
+Currently, the TTL feature has the following limitations:
+
+* The TTL attribute cannot be set on temporary tables, including local temporary tables and global temporary tables.
+* Tables with the TTL attribute do not support being referenced by other tables as the primary table in a foreign key constraint.
+* It is not guaranteed that all expired data is deleted immediately. The time when expired data is deleted depends on the scheduling interval and scheduling window of the background cleanup job.
+* Currently, a single table can only run a cleanup job on a single TiDB server node at a given time. This might cause performance bottlenecks in some scenarios (for example, when the table is extremely large). This issue will be optimized in future releases.
```

Comment:
```suggestion
* Currently, a single table can only run a cleanup job on a single TiDB node at a given time. This might cause performance bottlenecks in some scenarios (for example, when the table is extremely large). This issue will be optimized in future releases.
```